composer.json

```
@@ -20,32 +20,33 @@
         }
     ],
     "require": {
-        "php": ">=7.0",
         "az-digital/arizona-bootstrap": "2.0.5",
```

With automatic package sorting enabled (see comment near bottom of this file), the `php` version constraint still remains at the top of the list so we should probably leave the php line where it is.

```suggestion
        "php": ">=7.0",
        "az-digital/arizona-bootstrap": "2.0.5",
```